python/pyspark/sql/context.py

```diff
@@ -415,10 +522,10 @@ def createExternalTable(self, tableName, path=None, source=None, schema=None, **
         -------
         :class:`DataFrame`
         """
-        return self.sparkSession.catalog.createExternalTable(
+        return self.sparkSession.catalog.createExternalTable(  # type: ignore[attr-defined]
```

Comment:
If `createExternalTable` is missing from stubs it would be better to add it there (we should probably make a fix for the sake of 3.1 and 3.2 anyway) or wait for SPARK-36881 to be merged.

Reply:
__Note__ Created SPARK-36909 to track this.